velox/functions/prestosql/tests/ArraySortTest.cpp

```diff
@@ -583,6 +542,21 @@ TEST_F(ArraySortTest, lambda) {
       "(x, y) -> if(length(x) < length(y), 1, if(length(x) = length(y), 0, -1))");
 }
 
+TEST_F(ArraySortTest, failOnMapTypeSort) {
+  auto data = makeRowVector({makeAllNullMapVector(10, INTEGER(), VARCHAR())});
+  auto testFail = [&](const std::string& name, const std::string& lambdaExpr) {
+    VELOX_ASSERT_THROW(
+        evaluate(fmt::format("{}(c0, {})", name, lambdaExpr), data),
+        "(1 vs. 0)");
```

Comment:
Let's verify more of the error message. (1 vs. 0) is not very descriptive.

Reply:
I wonder whether we are testing the test framework here. Aren't there errors thrown while parsing the expression, not when trying to evaluate a function. If so, I wonder if it is better to just remove these tests.

Reply:
Yes, this test case is wrong, I am fixing it.

Reply:
I should use `makeAllNullMapVector` to create an ArrayVector before putting it into the RowVector.